tests/jobs.py

```diff
@@ -24,7 +24,9 @@
 from airflow import AirflowException, settings
 from airflow.bin import cli
 from airflow.jobs import BackfillJob, SchedulerJob
-from airflow.models import DagBag, DagRun, Pool, TaskInstance as TI
+from airflow.executors import DEFAULT_EXECUTOR
```

Comment:
Nit: Looks like these imports should be alpha sorted. We should probably add this rule to the lanscapebot.